### DISCUSSION TITLE
What happened to Supabase workflows?

### DISCUSSION THREAD

**Discussion by Mochael** [supabase/supabase#14133] [Questions]
Supabase workflows was announced 2 years ago [here](https://supabase.com/blog/supabase-workflows), but there haven't been any updates on it since. It was meant to compete with AWS step functions.

The [github repo](https://github.com/supabase/workflows) for it seems to be pretty inactive, did Supabase stop developing it?

**Comment by duneatnight** [comment]
Interested as well. We need workflows like yesterday
